src/components/BookList/BookList.jsx

```diff
@@ -0,0 +1,9 @@
+const BookList = () => (
+  <div className="books-page">
+    <button type="button" className="add-book-btn">Add Book</button>
+    <button type="button" className="remove-book-btn">Remove Book</button>
+    <ul />
+  </div>
```

Comment:
The project requirements tell to render 2 pages with no style that WILL show a list of books, therefore is not an immediate requirement. I will do it anyway since it will be needed in the future though